batavia/VirtualMachine.js

```diff
@@ -156,7 +174,11 @@ batavia.builtins = {
     hasattr: function() { throw new batavia.builtins.NotImplementedError("Builtin Batavia function 'hasattr' not implemented"); },
     hash: function() { throw new batavia.builtins.NotImplementedError("Builtin Batavia function 'hash' not implemented"); },
     help: function() { throw new batavia.builtins.NotImplementedError("Builtin Batavia function 'help' not implemented"); },
-    hex: function() { throw new batavia.builtins.NotImplementedError("Builtin Batavia function 'hex' not implemented"); },
+    hex: function(args) { 
+        if (args.length !== 1)
+            throw new batavia.builtins.TypeError("hex() takes exactly one argument (" + args.length + " given)");
+	return args[0].toString(16);
```

Comment:
do you need to prepend `0x`?

```
>>> hex(1)
'0x1'
```

vs.

```
> (1).toString(16)
'1'
```

Same goes for `oct` with a leading `0`.